mlflow/tensorflow/__init__.py

```diff
@@ -395,12 +491,15 @@ def load_model(model_uri, dst_path=None):
                      This directory must already exist. If unspecified, a local output
                      path will be created.
 
+    :param kwargs: kwargs to pass to ``keras.models.load_model`` method. Only available
+                   when you are loading a Keras model.
```

Comment:
Do we need an argument for specifying options for `tensorflow.saved_model.load`.

Reply:
Good question, I think we can add `saved_model_load_kwargs` for `load_model` ?

Reply:
Do we also need a separate argument for keras? I think that using the same names as we do for `save_model()` is good for consistency. 

```
        saved_model_kwargs=None,
        keras_model_kwargs=None
```